src/components/atoms/Calendar/Calendar.pcss

```diff
@@ -0,0 +1,76 @@
+.calendar {
+  text-align: center;
+  padding: 1rem;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  background-color: white;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
+
+  @media (--mq-from-medium) {
+    margin-left: 3rem;
+    margin-right: 3rem;
+  }
+
+  @media (--mq-from-large) {
+    margin-left: 5rem;
+    margin-right: 5rem;
+  }
+
+  @media (--mq-up-to-medium) {
+    .days-of-week {
+      font-size: 0.8rem; /* Decrease font size */
+    }
+
+    .day-of-week {
+      padding: 0.25rem;
+      white-space: nowrap; /* Prevent day names from wrapping */
+      overflow: hidden; /* Hide overflow text */
+      text-overflow: ellipsis; /* Show ellipsis for truncated text */
+    }
+
+    .calender-month-and-year {
+      font-size: 1rem;
+    }
+  }
+}
+
+.calendar-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.days-of-week {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  font-weight: bold;
+  margin-bottom: 0.5rem;
+}
+
+.day-of-week {
+  padding: 0.5rem;
+}
+
+.days {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+}
+
+.day {
+  padding: 0.5rem;
+  border: 1px solid #ccc;
+  transition: background-color 0.3s ease;
+}
+
+.day.available {
```

Comment:
`.calendar__days__day--available`

Dit is een modifier class, dus mag genest worden in zijn parent. Vb:

```
.calendar__days__day {
    // ...

    &--available {
        // ...
    }

    &--unavailable {
        // ...
    }
}
```